kubernetes/README.md

```diff
@@ -4,70 +4,111 @@ This example explains how to deploy the 1Passwrd SCIM bridge on Kubernetes runni
 
-The 1Password SCIM bridge requires SSL/TLS in order to communicate with your IdP. You must create a DNS record that points to your Kubernetes load balancer. This is a chicken and egg problem, as we need the load balancer before we can create the record. Please follow all of the steps until the load balancer has been created, then create your DNS record, but _do not attempt to perform a provisioning sync before the DNS records have been propogated_. The record must exist and the SCIM Bridge server must be running in order for LetsEncrypt to issue a certificate.
 
 ## Prepare your 1Password Account
 
-Log in to your 1Password account [using this link](https://start.1password.com/settings/provisioning/setup).  It will take you to the setup page for the SCIM bridge.
+Log in to your 1Password account [using this link](https://start.1password.com/settings/provisioning/setup). It will take you to the setup page for the SCIM bridge. Follow the instructions there.
+
+*IMPORTANT*: When the setup asks you for an email address for the new Provision Manager user it creates for you automatically, use a dedicated email address (for example: `op-provision-manager@example.com`) to handle this account. It is _not advised_ to use any personal email address!
+
+*IMPORTANT*: DO NOT create the Provision User manually. Let the setup process create the Provision Manager user for you **automatically.**
+
+Once complete, you'll be provided with two separate secrets:
+
+* a `scimsession` file 
+* a bearer token
+
+The `scimsession` file contains the credentials for the new Provision Manager user the setup process automatically created for you. This user will create, confirm, and suspend users, and create and manage access to groups.  
 
-Follow the on-screen instructions which will guide you through the following steps:
+*IMPORTANT*: Do not share these secrets! The bearer token must be provided to your Identity Provider, but beyond that it should be kept safe and not shared with anyone. Especially important is the `scimsession` file - do not share that with ANYONE, except the SCIM bridge.
+
+To reiterate: you'll need to share the bearer token with your Identity Provider, but it's important to **never share it with anyone else**. And never share your `scimsession` file with **anyone at all**.
+
+## Create your `scimsession` Kubernetes secret
+
+Next, we must create a Kubernetes secret containing the scimsession file. Using `kubectl`, we can read the scimsession file and create the secret in one command:
+
+```
+kubectl create secret generic scimsession --from-file=./scimsession
+```
+
+Make sure to pass the filepath of the scimsession file that you downloaded.  The above command will look for the file in  this folder (the `/kubernetes/` folder) of the repository.
 
-* Create a Provision Managers group
```

Comment:
I miss this initial list of items. Essentially with the list we are giving a set of acceptance criteria to determine if they have set it up correctly. I think the README benefited from it.